### PR TITLE
[DT-219][risk=no] Split review script into domain functions

### PR DESCRIPTION
**Pull request by dolbeew**

### CODE REVIEW

File: api/db-cdr/generate-cdr/build-review-all-events.sh [hunk 1]
```diff
@@ -6,7 +6,10 @@ set -e
 
 export BQ_PROJECT=$1   # project
 export BQ_DATASET=$2   # dataset
+export DATA_BROWSER=$3 # data browser flag
```
Comment (freemabd) [inline api/db-cdr/generate-cdr/build-review-all-events.sh:9]: This needs to be removed. The conditional workflows will take care of this.

File: api/db-cdr/generate-cdr/build-review-all-events.sh [hunk 2]
```diff
@@ -276,4 +293,32 @@ LEFT JOIN \`$BQ_PROJECT.$BQ_DATASET.concept\` c1 on a.PROCEDURE_CONCEPT_ID = c1.
...
-JOIN \`$BQ_PROJECT.$BQ_DATASET.person\` p on a.PERSON_ID = p.PERSON_ID"
\ No newline at end of file
+JOIN \`$BQ_PROJECT.$BQ_DATASET.person\` p on a.PERSON_ID = p.PERSON_ID"
+}
+
+if [[ "$DATA_BROWSER" == true ]]
+then
+  echo "Building index for data browser. Skipping creation of the cb_review_all_events table."
+  exit 0
+fi
```
Comment (freemabd) [inline api/db-cdr/generate-cdr/build-review-all-events.sh:303]: This is no longer needed cause of the conditional workflows.

**Review by freemabd — COMMENTED** [review]
Few change requests

File: api/db-cdr/generate-cdr/build-review-all-events.sh (hunk 1 above)
Comment (dolbeew) [reply api/db-cdr/generate-cdr/build-review-all-events.sh:9]: Good to know, done.

File: api/db-cdr/generate-cdr/build-review-all-events.sh (hunk 2 above)
Comment (dolbeew) [reply api/db-cdr/generate-cdr/build-review-all-events.sh:303]: Done.

**Review by freemabd — APPROVED** [review]

**Pull request merged**